default.nix

```diff
@@ -106,9 +106,9 @@ rec {
   terraform_1_0_0 = builtins.trace "terraform_1_0_0 is deprecated use terraform_1" terraform_1;
   terraform_1_0 = builtins.trace "terraform_1_0 is deprecated use terraform_1" terraform_1;
 
-  ansible = pkgs.ansible_2_10;
+  ansible = pkgs.ansible;
```

Comment:
No need to change this as nixpkgs is not updated

Reply:
Removed commit